akka-projection-jdbc/src/main/scala/akka/projection/jdbc/scaladsl/JdbcHandler.scala

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.jdbc.scaladsl
+
+import akka.projection.jdbc.JdbcSession
+import akka.projection.scaladsl.HandlerLifecycle
+
+trait JdbcHandler[Envelope, S <: JdbcSession] extends HandlerLifecycle {
+
+  def process(session: S, envelope: Envelope): Unit
```

Comment:
steal some docs from `Handler`

Reply:
Thou shalt not steal. 

But I will do it once while nobody is looking...